ansible_collections/f5networks/f5_modules/plugins/modules/bigip_pool_member.py

```diff
@@ -19,7 +19,7 @@
 short_description: Manages F5 BIG-IP LTM pool members
 description:
   - Manages F5 BIG-IP LTM pool members via iControl SOAP API.
-version_added: 1.4
+version_added: "1.4"
```

Comment:
this is one of the older modules it looks like been added into 1.4 ansible, so change this to 1.0